frontend/packages/dev-console/src/components/topology2/TopologyPage.tsx

```diff
@@ -86,16 +92,31 @@ const TopologyPage: React.FC<Props> = ({
         variant={showListView ? NamespacedPageVariants.light : NamespacedPageVariants.default}
         hideApplications={showListView}
         toolbar={
-          <Tooltip position="left" content={showListView ? 'Topology View' : 'List View'}>
-            <Link
-              className="pf-c-button pf-m-plain"
-              to={`/topology/${namespace ? `ns/${namespace}` : 'all-namespaces'}${
-                showListView ? '' : '/list'
-              }`}
-            >
-              {showListView ? <TopologyIcon size="md" /> : <ListIcon size="md" />}
-            </Link>
-          </Tooltip>
+          <div style={{ display: 'flex' }}>
+            {!showListView && !allProjectsView && (
+              <Popover
+                aria-label="Shortcuts"
+                bodyContent={TopologyShortcuts}
+                position="left"
+                maxWidth="50rem"
```

Comment:
This may as well be set to because we know we don't want any wrapping.
```suggestion
                maxWidth="100vw"
```